scripts/scil_split_image.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Splits the DWI image at certain indices along the last dimension (b-values).
+Many indices can be given at once by specifying multiple values. The splited
+volumes are in the same order as in the original file. Also outputs the
+corresponding .bval and .bvec files.
+
+This script can be useful for splitting images at places where a b-value
+extraction does not work. For instance, if one wants to split the x first
+b-1500s from the rest of the b-1500s in an image, simply put x as an index.
+
+"""
+
+import argparse
+import logging
+from pathlib import Path
+
+from dipy.io import read_bvals_bvecs
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, add_verbose_arg,
+                             assert_inputs_exist, assert_outputs_exist)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_dwi',
+                   help='The DW image file to split.')
+
+    p.add_argument('in_bval',
+                   help='The b-values file in FSL format (.bval).')
+
+    p.add_argument('in_bvec',
+                   help='The b-vectors file in FSL format (.bvec).')
+
+    p.add_argument('split_indices', nargs='+', type=int,
+                   help='The list of indices where to split the image. For '
+                        'example 3 10. This would split the image in three '
+                        'parts, such as [:3], [3:10], [10:].')
+
+    p.add_argument('--out_dwi', nargs='+', default=[],
```

Comment:
Would it be better to simply list the basename and the script automatically save nii.gz & bval & bvec?
@arnaudbore what do you think. These extension should be enforced anyway, right?

Reply:
You mean give a mandatory out_basename and automatically use my way of saving with the indices? (ex. dwi_0_3.nii.gz)

Reply:
I mean you list --out_basename dwi_0_0 dwi_0_1500 dwi_1_0 dwi_1_1500 or whatever

Then you know its gonna be .nii.gz .bval and .bvec
Why list 3 times the same thing if you know the extension already

Reply:
Ok yes. Should I assume it's .nii.gz, .bval and .bvec, or should I pick the same extensions as the input files? Sometimes, its .nii, .bvecs, .bvals...